datameta/api/openapi.yaml

```
@@ -189,6 +190,43 @@ paths:
         '500':
           description: Internal Server Error
 
+  /rpc/update-metadatasets:
```

```suggestion
  /rpc/replace-metadatasets:
```
Let's use a more explicit name